source/navier_stokes_matrix.cc

```diff
@@ -918,14 +925,14 @@ NavierStokesMatrix<dim>::local_divergence(
   FEEvaluation<dim, degree_p == -1 ? -1 : (degree_p + 1), degree_p + 2, dim> velocity(
     data, dof_index_u, quad_index_u);
   FEEvaluation<dim, degree_p, degree_p + 2, 1> pressure(data, dof_index_p, quad_index_u);
-
-  const VectorizedArray<double> *mu_values =
+  const VectorizedArray<double> *              mu_values =
```

Comment:
I used clang v11.1.0 for formatting. I'll have a look how to avoid this ugly formatting.

Reply:
Why is the indentation test not complaining?